Mutagen.Bethesda.Analyzers.Skyrim/Record/Npc/TrainerAnalyzer.cs

```diff
@@ -7,20 +7,35 @@ namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Npc;
 
 public class TrainerAnalyzer : IContextualRecordAnalyzer<INpcGetter>
 {
-    public static readonly TopicDefinition TrainerFactionMissingScript = MutagenTopicBuilder.DevelopmentTopic(
-            "Trainer requires script",
+    public static readonly TopicDefinition TrainerFactionMissingScript = MutagenTopicBuilder.FromDiscussion(
+            196,
+            "Trainer requires TrainerGoldScript",
             Severity.Warning)
-        .WithoutFormatting("Npc is a trainer but does not have a TrainerGoldScript");
+        .WithoutFormatting("Trainer npc does not have a TrainerGoldScript");
 
-    public static readonly TopicDefinition TrainerScriptMissingFaction = MutagenTopicBuilder.DevelopmentTopic(
-            "Trainer requires faction",
+    public static readonly TopicDefinition TrainerScriptMissingFaction = MutagenTopicBuilder.FromDiscussion(
+            197,
+            "Trainer requires trainer faction",
             Severity.Warning)
-        .WithoutFormatting("Npc has TrainerGoldScript but is not in a trainer faction");
+        .WithoutFormatting("Trainer npc is not in a trainer faction");
 
-    public static readonly TopicDefinition TrainerWithoutSpecialization = MutagenTopicBuilder.DevelopmentTopic(
+    public static readonly TopicDefinition TrainerWithoutSpecialization = MutagenTopicBuilder.FromDiscussion(
+            198,
             "Trainer without specialization",
             Severity.Warning)
-        .WithoutFormatting("NPCs with a JobMerchant faction need specialized Job faction, otherwise their merchant dialogue might not work");
+        .WithoutFormatting("Trainer npc does not have a specialized trainer faction");
+
+    public static readonly TopicDefinition TrainerWithLevel1 = MutagenTopicBuilder.FromDiscussion(
+            199,
+            "Trainer npc has level 1",
+            Severity.Warning)
+        .WithoutFormatting("Trainer npc has level 1, they won't be able to train you");
+
+    public static readonly TopicDefinition TrainerWithLevel1Min = MutagenTopicBuilder.FromDiscussion(
+            199,
+            "Trainer npc has min level 1",
+            Severity.Warning)
+        .WithoutFormatting("Trainer npc has min level 1, they won't be able to train you");
```

Comment:
Duplicate?  Why two topics with the same ID?

Reply:
They are so similar that I created a combined topic for them but I can split it again

Reply:
Done